docs/src/main/docbook/migration.xml

```diff
@@ -28,6 +28,18 @@
          xml:id="migration">
     <title>Migration Guide</title>
 
+    <section xml:id="mig-2.40">
+        <title>Migrating from Jersey 2.40 to 2.41</title>
+        <section xml:id="mig-2.40-breaking-changes">
+            <title>Changes in Jackson</title>
+            <para>
+                Jersey 2.41 starts to support Jackson 15 which comes with default limitations for the length of parsed text,
```

Comment:
again, no Jackson v15 has ever been released

Reply:
Right, it is 2.15